modules/silicon_design/scripts/build/notebooks/digilent_arty.repl

```diff
@@ -0,0 +1,24 @@
+using "digilent_arty_generated.repl"
```

Comment:
Do you need to commit those? How do they get included in the notebook environment?